### ISSUE TITLE
Enhancement - Ability to Drill into a DT Dashboard when selecting a "Pinned" SLO on a dashboard

### ISSUE THREAD

**Issue by stedelen** [dynatrace-oss/DynatraceDashboardPowerups#126]
Request -->When you select the SLO (pinned on a dashboard) instead of directing you to the SLO dashboard page (filtered to that specific SLO), you are directed to another dashboard? Use case = We have a main "summary SLO" that is a summary of steps in a customer journey. We have smaller (children) SLOs for each of the individual sub-steps.  When you select the main SLO, we would like to see a dashboard that features a breakdown of the child SLOs.

Knowing that there is a PU for images, which enables you to "click-to-dashboard" (example of an internal dashboard below).  
!PU(image):url=https://inspirewellness.sharepoint.com/sites/psm/SiteAssets/SRE%20Logo.jpg?raw=true;out=#dashboard;id=a2426c14-2261-4695-9fa0-7c368339ae7d

....Could we create a PU similar for SLOs?

!PU(SLO):url=https://dt.humana.com/e/d3610855-748f-4365-9bb1-f036451fac27/ui/slo?dashboardId=91a9cf8e-f851-4a95-a423-11e97ac0d810&id=38a8ed34-119f-3bb2-b3cb-55a0d439d4a2?raw=true;out=#dashboard;id=a2426c14-2261-4695-9fa0-7c368339ae7d

**Comment by LucasHocker** [comment]
Have you considered using !PU(menu)? You could have multiple drilldowns that way. John has been using this a lot in his UserJourney dashboard to jump to other types of analysis:
![Screen Shot 2021-11-05 at 2 25 37 PM](https://user-images.githubusercontent.com/34926264/140567361-600c2e66-2c55-4ead-a553-40b89b47e53e.png)

**Comment by stedelen** [comment]
@LucasHocker 

What would the Syntax look like, if applied to an SLO?

Apdex !PU(menu):name=Application;url=#uemapplications/uemappmetrics;uemapplicationId=APPLICATION-008569CDB300AE03 !PU(menu):name=Easy_Travel;url=http://www.easytravel.com

**Comment by LucasHocker** [comment]
Hmm... it doesn't look like there's a place to slot in markup for SLO tiles. I'll noodle how to make it cleaner, but for now you could put a markdown tile next to the SLO tile.

**Comment by LucasHocker** [comment]
Ok, I came up with a solution for this one. I added a parameter to the `menu` powerup for `title`. You'll use this to match the title of your SLO tile. So create a markdown tile and add the menu powerup syntax there. It should add the menu items to your SLO tile and hide the markdown tile
![Screen Shot 2021-11-08 at 9 50 32 AM](https://user-images.githubusercontent.com/34926264/140774659-35d4ab8a-807f-4cef-92e2-5ca39ca5168a.png)

Pushed enhancement to GitHub, adding to ChromeStore 1.58.
.
